Remotion/Web/Development.WebTesting/Configuration/WebTestConfigurationSection.cs

```diff
@@ -298,5 +298,12 @@ public bool Headless
     {
       get { return (bool)this[_headless]; }
     }
+
+    IWebTestChromiumSettings IWebTestSettings.Chrome => Chrome;
+
+    IWebTestChromiumSettings IWebTestSettings.Edge => Edge;
+
+    IWebTestHostingSettings IWebTestSettings.Hosting => new WebTestHostingAdapter();
+    IWebTestSiteLayoutSettings IWebTestSettings.TestSiteLayout => TestSiteLayoutConfiguration;
```

Comment:
Since we have the required instance of `Hosting` on `this` I think it would be better if we pass it into the adapter instead of the adapter resolving it. This allows us to remove the `WebTestConfigurationSection.Current` property afterwards. Please check if we can remove it, we should not have any more references to it.

```suggestion
    IWebTestChromiumSettings IWebTestSettings.Chrome => Chrome;

    IWebTestChromiumSettings IWebTestSettings.Edge => Edge;

    IWebTestHostingSettings IWebTestSettings.Hosting => new WebTestHostingAdapter(Hosting);

    IWebTestSiteLayoutSettings IWebTestSettings.TestSiteLayout => TestSiteLayoutConfiguration;
```

Reply:
We do need it just for the hosting settings as the original type was `ProviderSettings`, which is a .NET type. As such, we can't add our own interface on it like we do with the other config element types.

Reply:
Ah, thanks.